src/flux/app-config.coffee

```diff
@@ -4,11 +4,16 @@ flux = require 'flux-react'
 
 {makeSimpleStore} = require './helpers'
 
+MinimizedPattern = /tutor\.min-\d+/
```

Comment:
Great idea!  I've tested it out and works great under production.  It doesn't work in development on either port 3001 or 8000.  

What happens in development is the css is loaded from the /dist folder, i.e. `http://localhost:8000/dist/tutor.css`, which causes the url for images to also get the /dist prefix prepended.  
As an example the skyline image is:
`http://localhost:8000/dist/learning-guide/houston-skyline.svg` 
But it should be: 
`http://localhost:8000/resources/images/learning-guide/houston-skyline.svg`

That's an easy fix though, I'm just adding a `.replace("dist", "resources/images")`